consumer.go

```diff
@@ -595,6 +596,16 @@ func (child *partitionConsumer) parseResponse(response *FetchResponse) ([]*Consu
 	child.fetchSize = child.conf.Consumer.Fetch.Default
 	atomic.StoreInt64(&child.highWaterMarkOffset, block.HighWaterMarkOffset)
 
+	abortedProducerIDs := make(map[int64]none, len(block.AbortedTransactions))
```

Comment:
nitpicky, but I don't like none type, `struct{}` is more idiomatic. but none has been there from some time and might need a big refactor to get rid of.